components/dashboard/src/projects/Projects.tsx

```diff
@@ -122,19 +139,36 @@ export default function () {
                                 </div>
-                                <p>{toRemoteURL(p.cloneUrl)}</p>
+                                <a href={p.cloneUrl.replace(/\.git$/, '')}>
+                                    <p className="hover:text-gray-600 dark:hover:text-gray-400 dark:text-gray-500">{toRemoteURL(p.cloneUrl)}</p>
+                                </a>
+                            </div>
+                            <div className="h-10 px-6 py-1 text-gray-400 text-sm">
+                                <span className="hover:text-gray-600 dark:hover:text-gray-300">
+                                    <Link to={`/${teamOrUserSlug}/${p.name}`}>
+                                        Branches
+                                    </Link>
+                                </span>
+                                <span className="mx-2 my-auto">·</span>
+                                <span className="hover:text-gray-600 dark:hover:text-gray-300">
+                                    <Link to={`/${teamOrUserSlug}/${p.name}/prebuilds`}>
+                                        Prebuilds
+                                    </Link>
+                                </span>
                             </div>
-                            <div className="h-1/4 px-6 py-1"><p>__ Active Branches</p></div>
                         </div>
-                        <div className="h-1/6 px-6 border rounded-b-xl dark:border-gray-800 bg-gray-200 cursor-pointer" onClick={() => viewAllPrebuilds(p)}>
+                        <div className="h-10 px-4 border rounded-b-xl dark:border-gray-800 bg-gray-100 border-gray-100 dark:bg-gray-800">
                             {lastPrebuilds.get(p.id)
-                                ? (<div className="flex flex-row space-x-3 h-full text-sm">
-                                    <div className={"my-auto rounded-full w-3 h-3 text-sm align-middle " + (true ? "bg-green-500" : "bg-gray-400")}>
-                                        &nbsp;
-                                    </div>
-                                    <div className="my-auto">{lastPrebuilds.get(p.id)!.branch}</div>
-                                    <div className="my-auto text-gray-400">{moment(lastPrebuilds.get(p.id)!.startedAt, "YYYYMMDD").fromNow()}</div>
-                                    <div className="my-auto text-gray-400 flex-grow text-right">View All ⟶</div>
+                                ? (<div className="flex flex-row h-full text-sm justify-between">
+                                    <Link to={`/${teamOrUserSlug}/${p.name}/${lastPrebuilds.get(p.id)!.id}`} className="flex my-auto group space-x-2">
+                                        <img className="h-3 w-3 my-auto" src={getPrebuildStatusIcon(lastPrebuilds.get(p.id)!.status)} />
+                                        <div className="my-auto font-semibold text-gray-500 truncate w-24" title={lastPrebuilds.get(p.id)!.branch}>{lastPrebuilds.get(p.id)!.branch}</div>
```

Comment:
Nit: Maybe make add `dark:text-gray-400`?

<table><tr><th>BEFORE</th><th>AFTER</th></tr><tr><td>
<img width="374" alt="Screenshot 2021-08-25 at 18 52 54" src="https://user-images.githubusercontent.com/599268/130832807-c344f45b-a62c-49df-b6b6-0600c43bbf40.png">
</td><td>
<img width="370" alt="Screenshot 2021-08-25 at 18 54 29" src="https://user-images.githubusercontent.com/599268/130832813-55997343-594b-491d-959b-06fa69ee9204.png">
</td></tr></table>


```suggestion
                                        <div className="my-auto font-semibold text-gray-500 dark:text-gray-400 truncate w-24" title={lastPrebuilds.get(p.id)!.branch}>{lastPrebuilds.get(p.id)!.branch}</div>
```